service_mesh/service_mesh_install/prepare-to-deploy-applications-ossm.adoc

```diff
@@ -0,0 +1,30 @@
+[id="deploying-applications-ossm"]
+= Deploying applications on {ProductName}
+include::modules/ossm-document-attributes.adoc[]
+:context: deploying-applications-ossm
+
+toc::[]
+
+When you deploy an application into the {ProductShortName}, there are several differences between the behavior of applications in the upstream community version of Istio and the behavior of applications within a {ProductName} installation.
+
+.Prerequisites
+
+* Review xref:../service_mesh_install/understanding-ossm.html#ossm-vs-istio_understanding-ossm[Comparing {ProductName} and upstream Istio community installations]
+
+* Review xref:../service_mesh_install/installing-ossm.html#installing-ossm[Installing {ProductName}]
+
+include::modules/ossm-security-constraints.adoc[leveloffset=+1]
+
+include::modules/ossm-configure-security-constraints.adoc[leveloffset=+2]
+
+include::modules/ossm-master-configuration.adoc[leveloffset=+1]
+
+include::modules/ossm-updating-master-configuration.adoc[leveloffset=+2]
+
+include::modules/ossm-automatic-sidecar-injection.adoc[leveloffset=+3]
+
+include::modules/ossm-manual-sidecar-injection.adoc[leveloffset=+3]
+
+.Next steps
+
+* Deploy applications on {ProductName}.
```

Comment:
On the side menu, the next item is 'Removing service mesh'

Reply:
The Tutorials are in another PR right now, and the order of these items will likely move around after everything is assembled and published for TP12.